Gemfile

```diff
@@ -2,4 +2,4 @@ source "https://rubygems.org"
 
 gemspec
 
-gem "lita", path: "../lita"
+gem "lita", git: "https://github.com/litaio/lita.git"
```

Comment:
I have a feeling that this could cause problems, but I couldn't think of another way to fix the Travis build, which failed with Lita v4.7.0. Any ideas?